multus/multus.go

```diff
@@ -189,22 +189,30 @@ func delegateAdd(exec invoke.Exec, ifName string, delegate *types.DelegateNetCon
 			logging.Debugf("Set MAC address %q to %q", delegate.MacRequest, ifName)
 		}
 
-		if delegate.IPRequest != "" {
-			// validate IP address
-			if strings.Contains(delegate.IPRequest, "/") {
-				_, _, err := net.ParseCIDR(delegate.IPRequest)
-				if err != nil {
-					return nil, logging.Errorf("failed to parse CIDR %q", delegate.MacRequest)
+		if delegate.IPsRequest != nil {
+			var ips string
+			for _, ipRequest := range delegate.IPsRequest {
+				// validate IP address
+				if strings.Contains(ipRequest, "/") {
+					_, _, err := net.ParseCIDR(ipRequest)
+					if err != nil {
+						return nil, logging.Errorf("failed to parse CIDR %q", ipRequest)
+					}
+				} else if net.ParseIP(ipRequest) == nil {
```

Comment:
According to Dan's comment https://github.com/intel/multus-cni/pull/195#issuecomment-442312952 , there is a use case for an ip with no prefix.

Reply:
@SchSeba yeah, some plugins (host-local) already have the prefix in other configuration info, so they don't require it in the IPRequest (because it would be redundant for them). We also need to update the NPWG spec to clarify behavior around the prefix.

So for now I think we need to keep the prefix optional in Multus, but a given sub-plugin could fail if it doesn't get one.

Reply:
@dcbw Is this behaviour is clarified in the spec regarding the prefix ?